src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -43,6 +43,7 @@
     <!--Remove the Linux-specific code-->
     <Compile Remove="System\Device\Gpio\Drivers\Windows10Driver.notSupported.cs" />
     <Compile Remove="System\Device\I2c\I2cDevice.nonWindows.cs" />
+    <Compile Remove="System\Device\I2c\I2cBus.nonWindows.cs" />
```

Comment:
We should consider also removing UnixI2cBus, UnixI2cDevice and UnicI2cFileTransferBus from the Windows-specific build

Reply:
No need to do that on this PR though, we can do that later.

Reply:
Nope. `!= 'net5.0'` here actually means "net5.0-windows*". And this _should_ include the linux stuff, otherwise crosscompilling is no longer possible.